src/main/scala/cromwell/Boot.scala

```diff
@@ -0,0 +1,63 @@
+package cromwell
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+import akka.actor.{Props, ActorSystem}
+import akka.io.IO
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wordnik.swagger.model.ApiInfo
+import cromwell.engine.WorkflowManagerActor
+import cromwell.webservice._
+import spray.can.Http
+
+import scala.concurrent.duration._
+import scala.reflect.runtime.universe._
+import scala.util.{Failure, Success}
+
+
+object Boot extends App {
+
+  private def startup(): Unit = {
+    val conf = ConfigFactory.parseFile(new File("/etc/cromwell.conf"))
+
+    // we need an ActorSystem to host our application in
+    implicit val system = ActorSystem("cromwell")
+
+    val swaggerConfig = conf.getConfig("swagger")
+    val swaggerService = new SwaggerService(
+      swaggerConfig.getString("apiVersion"),
+      swaggerConfig.getString("baseUrl"),
+      swaggerConfig.getString("apiDocs"),
+      swaggerConfig.getString("swaggerVersion"),
+      Seq(typeOf[RootCromwellApiService]),
+      Option(new ApiInfo(
+        swaggerConfig.getString("info"),
+        swaggerConfig.getString("description"),
+        swaggerConfig.getString("termsOfServiceUrl"),
+        swaggerConfig.getString("contact"),
+        swaggerConfig.getString("license"),
+        swaggerConfig.getString("licenseUrl"))
+      ))
+
+    val service = system.actorOf(CromwellApiServiceActor.props(WorkflowManagerActor.props(system), swaggerService), "cromwell-service")
+
+    implicit val timeout = Timeout(5.seconds)
+    // start a new HTTP server on port 8080 with our service actor as the handler
+    import scala.concurrent.ExecutionContext.Implicits.global
+    (IO(Http) ? Http.Bind(service, interface = "0.0.0.0", port = 8080)).onComplete {
```

Comment:
I slightly disagree although I don't think it maters that much. We *needed* to do this in Zamboni because everything else depended on that particular call blocking until it was ready, but if we're starting from scratch it doesn't matter so much.

OTOH this is one of those cases where the anti-Await.result zealots are wrong